### DISCUSSION TITLE
How to upload a file to a storage bucket

### DISCUSSION THREAD

**Discussion by BigZhenjie** [supabase/supabase#31321] [Questions]
Hello guys! I am looking at the documentation and it does seem like storage is a supported feature. But I don't see anything related to uploading to a bucket...

Would I use insert?

**Comment by GaryAustin1** [comment]
You use the storage REST API and/or a client for your language.
https://supabase.com/docs/reference/javascript/storage-from-upload
https://supabase.com/docs/guides/storage/uploads/standard-uploads?queryGroups=language&language=js
